app.py

```diff
@@ -0,0 +1,12 @@
+from flask import Flask
```

Comment:
add Flask to requirements.txt

If you add new lib, please don't forget to add it to requirements.txt